faros_event.sh

```diff
@@ -298,7 +298,6 @@ function processArgs() {
     # No positional arg passed - show help
     if ! (($#)) || [ "$1" == "help" ]; then
         help
-        exit 0
```

Comment:
does `help` function exits with 0 already so it's redundant?

Reply:
That's correct. `shellcheck` 0.9.0 now displays an info notice about this which is why this is just now being flagged.